### DISCUSSION TITLE
DS18x20 after update

### DISCUSSION THREAD

**Discussion by olfprivat** [arendst/Tasmota#11191] [Q&A] [answered]
Sorry, but after update 9.2 ->9.3 I'm lost my settings for sensor DS18b20. Its invisible in GUI and by console..
Pls. Help me.

**Comment by nagyrobi** [comment, marked as answer]
What's your firmware flavor?
Maybe it was removed from there.
You should build your own flavor with the drivers you need. For example I needed a speciality to have DS18x20 in IR version, as I have an IR blaster in each room, equipped with an extra DS18B20. Was very easy to add DS18x20 and create a custom version just for me.

**Reply by barbudor** [reply]
If you are running `tasmota.bin(.gz)`, can you move to `tasmota-sensors.bin(.gz)` ?
There are some random issues with DS18x20 in `tasmota.bin(.gz)` which do not seems to occur in th e`-sensor` version.
Problem is still in qualification stage as the code hasn't been changed regarding DS18x20 between the 2 versions.

**Reply by sfromis** [reply]
See #11117

**Reply by tsunfinger** [reply]
same issue here on Sonoff Dual R2, but tasmota-sensors.bin fixed it

**Reply by sfromis** [reply]
It would appear to be fixed with newest dev release, without needing to use tasmota-sensors

**Reply by arendst** [reply]
See https://github.com/arendst/Tasmota/issues/11270#issuecomment-793905417

**Reply by arendst** [reply]
See #11270. New binaries have been released with v9.3.1
